lib/net.cpp

```diff
@@ -15,6 +15,25 @@
 
 // https://wiki.openssl.org/index.php/SSL/TLS_Client
 
+static std::string error_to_string(int err)
```

Comment:
This name is too generic - what kind of error? This is specifically related to sockets, so socket should be mentioned in the name somewhere.

Reply:
Changed name to socket_error_message()